rpms/SPECS/wazuh-manager.spec

```diff
@@ -328,6 +312,25 @@ if [ $1 = 1 ]; then
   %{_localstatedir}/packages_files/manager_installation_scripts/add_localfiles.sh %{_localstatedir} >> %{_localstatedir}/etc/ossec.conf
 fi
 
+# We create this fix for the operating system that decraped the SySV. For now, this fix is for suse/openSUSE
+sles=""
+  if [ -f /etc/SuSE-release ]; then
+    sles="suse"
+  elif [ -f /etc/os-release ]; then
+    if `grep -q "\"sles" /etc/os-release` ; then
+      sles="suse"
+    elif `grep -q -i "\"opensuse" /etc/os-release` ; then
+      sles="opensuse"
+    fi
+  fi
+
+  if [ ! -z "$sles" ] && [ $(ps --no-headers -o comm 1) == "systemd" ]; then
+    #If it's an upgrade and there is a service file in /etc/init.d/ we deleted.
```

Comment:
This comment is not accurate as this section will be executed in both install and upgrade.